### DISCUSSION TITLE
Aspiraciones en su carrera profesional

### DISCUSSION THREAD

**Discussion by efonsecab** [efonsecab/Mentoring-Blazor-y-mas#1] [General]
Favor contestar las siguientes preguntas

1. Cual es su experiencia actual con C#, y Blazor?
2. Cuales son sus aspiraciones en su carrera profesional, es decir, a que se quiere llegar a dedicar?

**Comment by kensolano** [comment]
1. Trabajo soportando aplicaciones en C# y he sido developer en sportsbook y empresas pequeñas.
2. Convertirme en full stack developer y conseguir un mejor empleo o de ser posible, emprender

**Comment by Gust23ccCastillo** [comment]
R-1/Mi experiencia con c# no ha sido en trabajos sino aprendiendo de la tecnología, se sobre las Apis Resfull en c#, programación orientada a objetos, implementación de cruds con apis , utilice blazor pero solo en proyectos pequeños de prueba los cuales fueron pequeños cruds,
desarrolle pequeñas aplicaciones en windows forms, utilización de .net core y entitiyframworks, conexiones de bd con sql server, y de todo un poco mas para ir conociendo el lenguaje.

R-2/ Mi aspiración  es abarcar bien el lenguaje y lograr dedicarme a una profecion de desarrollador backend

**Comment by EithanMM** [comment]
Respuestas:
1: Mi experiencia con C# ha sido principalmente en el desarrollo de APIs REST y SOAP que son consumidas por varios sistemas internos de la empresa, también he tenido la oportunidad de participar en el desarrollo y mantenimineto de varios microservicios que eran conumidos por un sistema completo, He manejado consultas en applicaciones utilizando tanto Entity Framework como Dapper como ORMs, actualmente me dedio a brindar soporte y mantenimineto a varias integraciones creadas con .NET Core  que son usadas a lo interno dentro de la empresa. Con respecto a Blazor tengo 0 experiencia desarrollando con dicha tecnología.

2: Mi aspiración es llegar a ser un Software Engineer Specialist, de ser posible con tecnologías de .NET o un tipo de Architect con énfasis en diseño de microservicios.

**Comment by JoseIgnacioChavarriaBrenes** [comment]
1. Mi experiencia en C# y Blazer son nulas, soy principiante.
2. Me encantaría dedicar a ser Fullstack o talvez dedicarme a desarrollo de videojuegos con C#

**Comment by Yahairacc** [comment]
Con C# lleve un curso en la universidad y de Blazer no tengo ningun conocimiento 
En cuanto a mis aspiraciones me gustaria trabajar como backend

**Comment by efrancotec** [comment]
1) Tengo poca experiencia laboral en el desrrollo con .Net, ya que soy QA
2) Me gustaría llegar a ser Fullstack especialista en .Net y también desarrollador de aplicaciones multiplataforma

**Comment by mardeanto** [comment]
1.Recibí un curso en la universidad de C# el otro no lo conozco.
2. fullstack
